packages/core/admin/admin/src/pages/Webhooks/ListView/index.js

```diff
@@ -267,56 +202,189 @@ function ListView() {
     }
   };
 
+  const handleSelectAllCheckbox = () => {
+    dispatch({
+      type: 'SET_ALL_WEBHOOKS_TO_DELETE',
+    });
+  };
+
+  const handleSelectOneCheckbox = (value, id) => {
+    dispatch({
+      type: 'SET_WEBHOOKS_TO_DELETE',
+      value,
+      id,
+    });
+  };
+
   const handleGoTo = to => {
     push(`${pathname}/${to}`);
   };
 
-  if (isLoading) {
-    return <LoadingIndicatorPage />;
-  }
-
   return (
-    <Wrapper>
+    <Layout>
       <PageTitle name="Webhooks" />
-      <Header {...headerProps} />
-      {canRead && (
-        <div className="list-wrapper">
-          {rowsCount > 0 ? (
-            <List
-              {...listProps}
-              customRowComponent={props => {
-                return (
-                  <ListRow
-                    {...props}
-                    canUpdate={canUpdate}
-                    canDelete={canDelete}
-                    onCheckChange={handleChange}
-                    onEditClick={handleGoTo}
-                    onDeleteCLick={handleDeleteClick}
-                    onEnabledChange={handleEnabledChange}
-                    itemsToDelete={webhooksToDelete}
-                  />
-                );
-              }}
-            />
-          ) : (
-            <EmptyState
-              title={formatMessage({ id: 'Settings.webhooks.list.empty.title' })}
-              description={formatMessage({ id: 'Settings.webhooks.list.empty.description' })}
-              link="https://strapi.io/documentation/developer-docs/latest/development/backend-customization.html#webhooks"
-              linkText={formatMessage({ id: 'Settings.webhooks.list.empty.link' })}
+      <Main labelledBy="webhooks">
+        {isLoading ? (
+          <LoadingIndicatorPage />
```

Comment:
Even when displaying the loader, I think the HeaderLayout should be on the page because else, during this tiny moment where nothing is displayed, there's nothing on the screen and the page is in an invalid state because there's not H1 on it.

Also, when loading something, you can put a `aria-busy={isLoading}` on the "Main" component so that it tells people using ATs that something is happening.

Later on, when the data has been loader, I think it's a common practice to use an aria region to announce that the content has been loaded and that the content of the Table is ready to be navigated 😊